dapp/src/hooks/useScrollbarVisibility.ts

```diff
@@ -0,0 +1,29 @@
+import { useCallback, useEffect, useState } from "react"
+
+const SCROLLBAR_WIDTH = `${window.innerWidth - document.body.offsetWidth}px`
+
+function scrollbarVisible(className: string) {
```

Comment:
Non blocking:
It's a common pattern to select DOM elements with selectors, not only class names so I'd use `querySelector` instead of `getElementsByClassName`. Also the name made me confused. I had to double check what the function does. What do you think?

```suggestion
function isScrollbarVisible(selector: string) {
```

Reply:
https://github.com/thesis/acre/pull/766/commits/f75fe59747eac984f19eab26303094e32128b078 https://github.com/thesis/acre/pull/766/commits/038a7f2e27d883b8f2a2fdb28a8a4a0fc3764b4b